modules/ldsg.js

```diff
@@ -0,0 +1,136 @@
+// @ts-check
+
+const Augur = require("augurbot-ts"),
+  u = require("../utils/utils"),
+  Discord = require("discord.js"),
+  { ButtonStyle } = require("discord.js");
+
+/**
+ * Responds with the number of guild members, and how many are online.
+ * @param {Discord.ChatInputCommandInteraction} interaction The interaction that the user submits.
+ */
+async function slashLdsgMembers(interaction) {
+  try {
+    const ldsg = interaction.client.guilds.cache.get(u.sf.ldsg);
+    if (!ldsg) throw new Error("Couldn't find LDSG");
+    const online = ldsg.members.cache.filter((member) => member?.presence && member.presence.status != "offline");
+    const response = `📈 **Members:**\n${ldsg.memberCount} Members\n${online.size} Online`;
+    await interaction.reply({ content: response });
+  } catch (error) { u.errorHandler(error, interaction); }
+}
+
+const sendOptions = [
+  u.MessageActionRow().setComponents([
+    new u.Button().setCustomId("suggestionPA").setEmoji("🗣️").setLabel("Public Affairs").setStyle(ButtonStyle.Success),
+    new u.Button().setCustomId("suggestionLog").setEmoji("🚚").setLabel("Logistics").setStyle(ButtonStyle.Success),
+    new u.Button().setCustomId("suggestionOps").setEmoji("🗓️").setLabel("Operations").setStyle(ButtonStyle.Success),
+    new u.Button().setCustomId("suggestionMgmt").setEmoji("⚙️").setLabel("Management").setStyle(ButtonStyle.Primary),
+    new u.Button().setCustomId("suggestionIcarus").setEmoji("🤖").setLabel("Icarus").setStyle(ButtonStyle.Primary),
+  ]),
+  u.MessageActionRow().setComponents([
+    new u.Button().setCustomId("suggestionIgnore").setEmoji("⚠️").setLabel("Ignore").setStyle(ButtonStyle.Danger),
+  ])
+];
+
+const replyOption = [
+  u.MessageActionRow().setComponents([
+    new u.Button().setCustomId("suggestionReply").setEmoji("🗨️").setLabel("Reply to user").setStyle(ButtonStyle.Primary)
```

Comment:
maybe add an option to rephrase the suggestion or identify the parts such as the issue, the root cause, and possible steps to take (with the original request still there, just not as dominant) with a modal